moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -263,3 +261,47 @@ def get_attached_objects(self, object_ids = []):
             conversions.msg_from_string(msg, ser_aobjs[key])
             aobjs[key] = msg
         return aobjs
+
+    def check_object_does_not_exist(self, object_name, timeout=4):
```

Comment:
The function names are not reflecting the fact that they actually wait for a specific event. 
```suggestion
    def wait_for_object_removed(self, object_name, timeout=4):
```